src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1395,6 +1398,14 @@ void RaftPart::processAskForVoteRequest(
             });
     }
 
+    if (!req.get_is_transfer() && req.get_committed_log_id() < committedLogId_) {
```

Comment:
commited mean the log has been write to the state machine, it is a "Durability" log, maybe we can not reset it directly.